### DISCUSSION TITLE
Website text typos

### DISCUSSION THREAD

**Discussion by chbrandt** [eclipse-edc/.github#140] [General]
I read some errors in the last paragraph in [https://projects.eclipse.org/projects/technology.edc](https://projects.eclipse.org/projects/technology.edc) and I don't know (1) where to PR the corrections nor (2) where to raise an issue, so I'll do it here.

As of now, the it reads:

> _"The EDC project is governed by the Eclipse Foundation and is and its project management, implementation, communication, and compliance follow the corresponding roles and rules, based on a mediocratic paradigmbased on the principles of openness, transparency, and meritocracy. The EDC Committers provides a core group of people that are ensures the project’s progress roadmap and give provides guidance to the community of contributors, users, and other stakeholders."_

I suggest the following (fixes):

> _"The EDC project is governed by the Eclipse Foundation. Its project management, implementation, communication, and compliance follow the corresponding roles and rules, based on the principles of openness, transparency, and meritocracy. The EDC Committers compose a core group of people that ensures the project's progress roadmap and give guidance to the community of contributors, users, and other stakeholders."_

**Comment by mspiekermann** [comment]
Thanks a lot for the hint :thumbsup:. 

Website is updated.
